axelor-stock/src/main/resources/views/Selects.xml

```diff
@@ -127,4 +127,9 @@
 		<option value="com.axelor.apps.production.db.OperationOrder" data-grid="operation-order-grid" data-form="operation-order-form">Operation order</option>
 	</selection>
 
+	<selection name="stock.location.wizard.export.type.select">
```

Comment:
don't define new selection but use 'iadministration.export.type.select' from base module. also use 'selection-in' attribute to allow only xls and pdf option.